pyelasticsearch/tests/client_tests.py

```diff
@@ -273,8 +279,8 @@ def test_bulk(self):
                     index='test-index',
                     doc_type='book')
         except BulkError as exc:
-            eq_(exc.successes, [])
-            eq_(exc.errors, [{'index': {'status': 409,
+            self.assertEqual(exc.successes, [])
+            self.assertEqual(exc.errors, [{'index': {'status': 409,
                                         '_type': 'book',
```

Comment:
We should fix the indentation in places like this.

Reply:
Fixed too.